app/src/main/java/com/piwniczna/mojakancelaria/activities/LoginActivity.kt

```diff
@@ -27,4 +33,38 @@ class LoginActivity : AppCompatActivity() {
         val myIntent = Intent(this, MainActivity::class.java)
         startActivityForResult(myIntent, 123)
     }
+
+    fun testDb(){
+        AsyncTask.execute {
+
+            //how to init db
+            try {
+                database = Room.databaseBuilder(
```

Comment:
please extract this later to separate file

Reply:
You can create an issue directly from this comment

Reply:
But there is already one issue about that so :P

Reply:
this one - #6  ?

Reply:
@sasuke5055

Reply:
Tak

Reply:
Ok,  thanks :p